File: src/main/java/com/vacaciones/payLoads/CreateRequestPayload.java
package com.vacaciones.payLoads;

import java.sql.Date;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class CreateRequestPayload {
    
    private String issue;
    private String state;
    private Date startDate;
    private Date endDate;
    private String nameTypeOfRequest;
    private String descriptionCauseRequest;
    private Integer quantityOfDays;
}

File: src/main/frontend/src/apiCall/payloads/CreateRequestPayload.js
export default class CreateRequestPayload {
    issue;
    state;
    startDate;
    endDate;
    nameTypeOfRequest;
    descriptionCauseRequest;
    quantityOfDays;

    constructor(issue,state,startDate,endDate,nameTypeOfRequest,descriptionCauseRequest,quantityOfDays){
        this.issue = issue
        this.state = state
        this.startDate = startDate
        this.endDate = endDate
        this.nameTypeOfRequest = nameTypeOfRequest
        this.descriptionCauseRequest = descriptionCauseRequest
        this.quantityOfDays = quantityOfDays 
    }
}